akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/EventSourcedBehavior.scala

```diff
@@ -41,8 +42,10 @@ object EventSourcedBehavior {
     persistenceId:  PersistenceId,
     emptyState:     State,
     commandHandler: (State, Command) ⇒ Effect[Event, State],
-    eventHandler:   (State, Event) ⇒ State): EventSourcedBehavior[Command, Event, State] =
-    EventSourcedBehaviorImpl(persistenceId, emptyState, commandHandler, eventHandler)
+    eventHandler:   (State, Event) ⇒ State): EventSourcedBehavior[Command, Event, State] = {
+    val loggerClass = LoggerClass.getLoggerClass(classOf[EventSourcedBehavior[_, _, _]])
```

Comment:
I think this should work with the javadsl EventSourcedBehavior, since its `apply` calls `scaladsl.EventSourcedBehavior.apply`, but might be worth testing?

Reply:
Yeah, pretty sure, but I'll double check.

Reply:
It didn't, so I changed it to get a good logger class.